hadoop-common-project/hadoop-common/src/main/resources/core-default.xml

```diff
@@ -2512,12 +2512,22 @@ The switch to turn S3A auditing on or off.
 </property>
 
 <property>
-    <name>ipc.server.log.slow.rpc</name>
-    <value>false</value>
-    <description>This setting is useful to troubleshoot performance issues for
-     various services. If this value is set to true then we log requests that
-     fall into 99th percentile as well as increment RpcSlowCalls counter.
-    </description>
+  <name>ipc.server.log.slow.rpc</name>
+  <value>false</value>
+  <description>This setting is useful to troubleshoot performance issues for
+   various services. If this value is set to true then we log requests that
+   fall into 99th percentile as well as increment RpcSlowCalls counter.
+  </description>
+</property>
```

Comment:
revert this code format modification.

Reply:
Thanks @ZanderXu  quick review, has been updated PR